exercises/sheet-03.md

```diff
@@ -0,0 +1,23 @@
+# Sheet 3: Profile Likelihood and bootstrapping
+
+In this sheet you will learn more about parameter estimation by constructing a profile likelihood ratio and using bootstrapping techniques for finding a confidence interval.
+
+You will continue working with the sample that you have obtained for Sheet 2.
+
+When fitting the $\Xi^- K^+ \pi^+$ invariant-mass distribution (`Lc_M`) for the previous sheet and inspecting the pulls carefully, you may have noticed a second peak at higher mass.<br>
+This is exciting! If you can prove that the small enhancement is statistically significant, you have evidence for a decay process that has never been measured: $\Xi_c^+ \rightarrow \Xi^- K^+ \pi^+$.<br>
+To work towards that, you will estimate the *yield* (i.e. the integral of the signal p.d.f.) of this process.
+
+## Exercise 1
+
+Add a second Gaussian to your fit model, fit the data again, and compute the Hessian matrix. Use that to compute the uncertainty of the signal yield for the new decay process.
```

Comment:
Ask also about parameter correlation. I suggest asking to plot the correlation matrix in color coding `heatmap`